eZ/Bundle/EzPublishCoreBundle/Resources/views/default/full.html.twig

```diff
@@ -0,0 +1,5 @@
+<h2>{{ ez_content_name(content) }}</h2>
+{% for field in content.FieldsByLanguage %}
```

Comment:
Good question.

Maybe it would work with a lowercase 'f', but this uses Twig's syntax that ignores the get from called methods. The method is called getFieldsByLanguage, so I kept the uyppercase 'F'. Maybe it doesn't matter, in which case we should change it.